classes/dpkg-bazel.bbclass

```diff
@@ -13,6 +13,17 @@ inherit dpkg
 
 DEPENDS += "bazel-bootstrap"
 
+# packages to install into the shared buildchroot
+BAZEL_INSTALL = "bazel-bootstrap git symlinks"
+
+# install bazel into the buildchroot so we can fetch external sources using bazel
+bazel_install() {
+    buildchroot_do_mounts
+    sudo -E chroot ${BUILDCHROOT_DIR} apt-get install -y ${BAZEL_INSTALL}
```

Comment:
that's clear, we however have a bootstrapping problem here as we can only enter the sbuild/schroot once we have all sources (and the debian/ folder) in particular

Reply:
Just give me some minutes. I'll try to unify both series. Then only the CCACHE issue remains, but maybe we will find a solution as well.

Reply:
Could you please check if `fm/coral-sbuild-v2` works for you. This branch contains your changes but takes my `do_fetch_bazel_deps` logic instead of yours (2b07aac323b25417173ff98d1d0f06480c67b57b instead of 1f367c4ddebde8842306b63fded7c5cd2dae731c)